include/isotp.h

```diff
@@ -0,0 +1,350 @@
+/*
```

Comment:
We are trying to migrate away from include files that are in `include/` so this could be placed to `include/can` or actually the `include/canbus` might be better as the "can" word is quite ambiguous.

Reply:
As I said, the isotp.h needs to go into a subdirectory under include/

Reply:
Done.

Reply:
Added my name to CODEOWNERS /include/canbus/